terraform/modules/self-service/variables.tf

```
@@ -0,0 +1,3 @@
+variable "deployment" {
+  description = "Name of the deployment; {joint,staging,prod,integration}"
```

joint doesn't exist anymore
```suggestion
  description = "Name of the deployment; {staging,prod,integration}"
```

---

Thanks @jakubmiarka have amended!